### DISCUSSION TITLE
Missing Confirmation Alert Box When Accidentally Moving Folder into Another Folder in Cyberduck

### DISCUSSION THREAD

**Discussion by weichung96** [iterate-ch/cyberduck#14887] [Q&A] [answered]
There seems to be a missing of confirmation prompt when a folder is mistakenly dragged and dropped into another folder.

**Comment by dkocher** [comment, marked as answer]
I cannot reproduce this on macOS running Version 8.6.2 (39964). You may have to reset the preference `browser.move.confirm` running 

```
defaults delete ch.sudo.cyberduck browser.move.confirm
```

on macOS.
